semantic-java/src/Language/Java/Tags.hs

```diff
@@ -153,8 +174,8 @@ instance ToTags Java.InferredParameters
 instance ToTags Java.InstanceofExpression
 instance ToTags Java.IntegralType
 instance ToTags Java.InterfaceBody
-instance ToTags Java.InterfaceDeclaration
-instance ToTags Java.InterfaceTypeList
+--instance ToTags Java.InterfaceDeclaration
```

Comment:
🔪

Reply:
I think we’re keeping these comments for clarity’s sake, so that people can just look at the alphabetized view of syntax nodes and, via the presence of comments, know which ones have custom definitions and which ones are boring.